### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by Perpee**
Please review my work @promise-J and @ChidiPatrick and @Pieta9221

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Pieta9221) [inline html/index.html]: Add a favicon

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        <link rel="stylesheet" href="../css/style.css" />
+        <title>Google clone</title>
+    </head>
+    <body>
+        <div class="container">
+            <header>
+                <a href="#">Gmail</a>
+                <a href="#">Images</a>
+                <img class="app" src="../images/apps_black_24dp.svg" alt="google app icon" />
```
Comment (Pieta9221) [inline html/index.html]: Should also be a link

File: html/index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                <img class="account" src="../images/account_circle_FILL0_wght400_GRAD0_opsz48.png"
```
Comment (Pieta9221) [inline html/index.html]: Make it a link

File: README.md [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
+
+##Visit my live site here https://perpe123.github.io/Lesson_1_and_2_Final_Task_Obiefuna_Toochi/
+
+##How to run/test this project
+
+##ensure you have git installed properly on your machine
+
+##ensure you have node js installed in your machine
+
+##open your terminal and run "git clone "
```
Comment (Pieta9221) [inline README.md]: Add link to repository

**Review by Pieta9221 — COMMENTED** [review]
:star2: :star2: :star2:
Good Job
Effect corrections and merge

File: html/index.html (hunk 2 above)
Comment (promise-J) [reply html/index.html]: Please what should be a link @Pieta9221

File: html/index.html (hunk 3 above)
Comment (promise-J) [reply html/index.html]: Again what do I make a link?

**Review by promise-J — APPROVED** [review]
You can merge your changes to the main branch.

**Pull request merged**